pkgs/applications/networking/instant-messengers/deltachat-desktop/default.nix

```diff
@@ -76,10 +76,6 @@ buildNpmPackage rec {
     VERSION_INFO_GIT_REF = src.rev;
   };
 
-  preBuild = ''
-    rm -r node_modules/deltachat-node/node/prebuilds
```

Comment:
Aren't there other prebuilds we should delete now? I think we have to use deltachat-rpc-server.

Reply:
I couldn't find any.